tuf/api/metadata.py

```diff
@@ -363,7 +363,9 @@ def verify_delegate(
 
         Raises:
             UnsignedMetadataError: 'delegate' was not signed with required
-                threshold of keys for 'role_name'
+                threshold of keys for 'role_name'.
+            ValueError: no delegation was found for 'delegated_role'.
+            TypeError: called this function on non-delegating metadata class.
```

Comment:
Please coordinate with #1815, where we start markup all variables with double-backtick.